src/components/Article-Page.jsx

```diff
@@ -0,0 +1,45 @@
+import axios from "axios";
+import { useState, useEffect } from "react";
+import { useParams, Link } from "react-router-dom";
+
+export default function ArticlePage() {
+  const { article_id } = useParams();
+  const [articleToDisplay, setArticleToDisplay] = useState([]);
+  useEffect(() => {
+    axios
+      .get(
+        "https://north-star-articles.onrender.com/api/articles/" + article_id
+      )
```

Comment:
You could extract out this axios call, also `${}` may be neater than string concatenation